usage/rest-api/src/main/java/brooklyn/rest/api/CatalogApi.java

```diff
@@ -228,5 +228,12 @@ public Response getIcon(
         @ApiParam(name = "version", value = "version identifier of catalog item (application, entity, policy)", required=true)
         @PathParam("version") String version);
 
+    @POST
+    @Path("/entities/{itemId}/deprecated/{deprecated}")
+    public void setDeprecated(
+        @ApiParam(name = "itemId", value = "The ID of the catalog item to be deprecated", required = true)
```

Comment:
Can we add a test of the REST api that uses the name:version format, as that is what users are more likely to want to do.